### PR TITLE
ci: github actions

### PR DESCRIPTION
**Pull request by grant**
Sets up GitHub Actions. Fixes https://github.com/GoogleCloudPlatform/functions-framework-php/issues/39

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+on: [push, pull_request]
+
+jobs:
+  run:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ubuntu-latest]
+        php-versions: ['7.1', '7.2', '7.3', '7.4']
```
Comment (bshaffer) [inline .github/workflows/ci.yaml:11]: We can drop 7.1 as this version of PHP is no longer supported, and App Engine Standard starts at 7.2

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: CI
```
Comment (bshaffer) [inline .github/workflows/ci.yaml:1]: nit, but let's call this `tests.yaml` to be consistent with other PHP repos

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (grant) [reply .github/workflows/ci.yaml]: Dropped 7.1.

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (grant) [reply .github/workflows/ci.yaml]: Renamed file to `.github/workflows/tests.yaml`

**Comment by grant** [comment]
Fixes applied. PTAL :)

**Pull request merged**